crates/air_r_formatter/src/r/auxiliary/call.rs

```diff
@@ -1,52 +1,104 @@
+use crate::comments_directives;
 use crate::is_suppressed_by_comment;
 use crate::prelude::*;
+use crate::r::auxiliary::call_arguments::FormatRCallArgumentsOptions;
+
 use air_r_syntax::AnyRExpression;
 use air_r_syntax::AnyRSelector;
 use air_r_syntax::RCall;
 use air_r_syntax::RCallFields;
 use air_r_syntax::RIdentifier;
+use air_r_syntax::RLanguage;
+use biome_formatter::FormatRuleWithOptions;
 use biome_formatter::write;
+use biome_rowan::AstNode;
 use biome_rowan::SyntaxResult;
+use comments::Directive;
+use comments::FormatDirective;
 use settings::Skip;
 
 #[derive(Debug, Clone, Default)]
-pub(crate) struct FormatRCall;
+pub(crate) struct FormatRCall {
+    table: bool,
```

Comment:
Would you mind making this `table: Option<bool>`? Imagine if a caller tried to set this option by explicitly supplying `table: false`. That would not work as expected, because `is_table_call()` would still run rather than just accepting what the caller explicitly provided.

I also find it hard to grok that `table = false` is the `Default` that we get when this is unset.